tests/cases/standalone/common/function/time.sql

```diff
@@ -1,3 +1,5 @@
 -- SQLNESS REPLACE (\d+:\d+:\d+\.\d+) TIME
 -- SQLNESS REPLACE [\s\-]+
 select current_time();
+
+SELECT to_timezone('2022-09-20T14:16:43.012345+08:00', 'Europe/Berlin');
```

Comment:
I am confusing how to pass int64 as argument?, I passed a number, got `StringType`?

Reply:
The sqlness will render the timestamp in the string form from the results.